File: oleg-peresada/service2/Controllers/ServiceController.fs
namespace service2.Controllers

open System
open System.Net.Http
open Microsoft.AspNetCore.Mvc
open Microsoft.Extensions.Logging

[<ApiController>]
[<Route("/")>]
type ServiceController (logger : ILogger<ServiceController>) =
    inherit ControllerBase()

    [<HttpGet>]
    member __.Get() : Async<String> =
        async {
            let http = new HttpClient()
            let! response = http.GetAsync(Environment.GetEnvironmentVariable("URL_SERVICE")) |> Async.AwaitTask
            let! content = response.Content.ReadAsStringAsync() |> Async.AwaitTask
            return if (content = "На") then content + " .Net" else "Ответ не соответствует ожидаемому"
        }

File: oleg-peresada/service1/Controllers/ServiceController.cs
using Microsoft.AspNetCore.Mvc;

namespace service1.Controllers 
{
    [ApiController]
    [Route("[controller]")]
    public class ServiceController : ControllerBase
    {
        [HttpGet("/")]
        public ActionResult Get() 
        {
            return Ok("На");
        }
    }
}